py_gtfs_rt_ingestion/py_gtfs_rt_ingestion/__init__.py

```diff
@@ -1,3 +1,5 @@
 __version__ = '0.1.0'
 
 from .configuration import Configuration
+from .config_base import ConfigType
+from .config_base import ConfigDetail
```

Comment:
Probably don't need ConfigDetail in here.

Reply:
yeah. i think it should be taken out, but there aren't really hard, accepted rules here.

i tend to think of it as "What stuff would I want to use outside of this library". it includes the added benefit of giving us flexibility to change the underlying structure, but keep the API consistent by changing where the imports are coming from. its not a big concern here given we're the only consumers and its unlikely we're going to even touch this module much after we get the lambda stuck kicking off.

my general feelings about `__init__.py` are pretty well captured in this old reddit thread.

https://old.reddit.com/r/Python/comments/1bbbwk/whats_your_opinion_on_what_to_include_in_init_py/